backend/app/services/adapters/task_kinds/operations.py

```
@@ -142,27 +148,79 @@ def _handle_existing_task(
                 detail="task already clear, please create a new task",
             )
 
-        # Check expiration
-        expire_hours = settings.APPEND_CHAT_TASK_EXPIRE_HOURS
+        # Get task type for error messages
         task_type = (
             task_crd.metadata.labels
             and task_crd.metadata.labels.get("taskType")
```

Ruff (0.14.14)

[warning] Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)